lib/api/core/plugins/pluginsManager.js

```diff
@@ -831,8 +831,15 @@ class PluginsManager {
  * @returns {Promise}
  */
 function triggerHooks(emitter, event, data) {
+  const wildcardEvents = getWildcardEvents(event)
+    // filter events like 'foo:*' because EventEmitter2 
```

Comment:
I think it would be simpler (and better) to completely get rid of EventEmitter2, since we're only using it for wildcards (this should be a light change since EventEmitter2 has the same interface than the native EventEmitter).

And instead manage wildcards ourselves entirely, because that kind of exception handling is a bit costly performances wide, and events triggering is a critical part of Kuzzle.

Reply:
Also, this code is unsafe because `getWildcardEvents` can return null.